lib/parity/environment.rb

```diff
@@ -130,7 +130,9 @@ def git_remote
     end
 
     def heroku_app_name
-      git_remote.split(":").last.split(".").first
+      git_remote.
```

Comment:
@gabebw thank you for this recommendation. I updated to use your suggested approach and I agree that it looks clearer.